server/testdata/integration/error_handler/04_couper.hcl

```diff
@@ -0,0 +1,70 @@
+server "scoped" {
+
+  access_control = ["secured"]
+
+  api {
+    base_path = "/api"
+
+    beta_scope = "read"
+    endpoint "/" {
+      response {
+        status = 204
+      }
+    }
+
+    endpoint "/pow/" {
+      beta_scope = {
+        post = "power"
+      }
+
+      response {
+        status = 204
+      }
+
+      error_handler "beta_insufficient_scope" {
+        response {
+          status = 400
+          body = "Not enough power"
+        }
+      }
+
+      error_handler "beta_operation_denied" {
+        response {
+          status = 405
+          body = "Not enough power"
+        }
+      }
+    }
+
+    error_handler "beta_scope" {
+      response {
+        status = 418
+      }
+    }
+
+  }
+
+  endpoint "/" {
```

Comment:
What's the purpose of this (non-API) endpoint?

Reply:
tests updated, used now